common/utils/utils.go

```diff
@@ -588,13 +588,13 @@ func CompareSchema(conv1, conv2 *internal.Conv) error {
 	for spannerTableInd := range conv1.SpSchema {
 		sessionTable := conv1.SpSchema[spannerTableInd]
 		spannerTable := conv2.SpSchema[spannerTableInd]
-		if sessionTable.Name != spannerTable.Name || sessionTable.Parent != spannerTable.Parent ||
-			len(sessionTable.Pks) != len(spannerTable.Pks) || len(sessionTable.ColDefs) != len(spannerTable.ColDefs) ||
+		if sessionTable.Name != spannerTable.Name || sessionTable.ParentId != spannerTable.ParentId ||
```

Comment:
We should move to using ids instead of comparing with names

Reply:
Updated. Please review it.